### ISSUE TITLE
Unknown Server Error (503); when listing nodes

### ISSUE THREAD

**Issue by shamil** [kubernetes/dashboard#2200]
##### Environment
```
Dashboard version: 1.6.2
Kubernetes version: 1.7.0 and 1.6.2
Operating system: Debian GNU/Linux 8.7 (jessie)
Heapster version: 1.3.0
```

##### Steps to reproduce

Navigate to Nodes (from sidebar).

##### Observed result

In browser

`Unknown Server Error (503)`

```
Error: 'EOF'
Trying to reach: 'http://172.18.0.4:9090/api/v1/node?filterBy=&itemsPerPage=15&namespace=&page=1&sortBy=d,creationTimestamp'
```

In container logs

```
[2017-07-27T21:22:08Z] Incoming HTTP/1.1 GET /api/v1/node?filterBy=&itemsPerPage=15&namespace=&page=1&sortBy=d,creationTimestamp request from 10.103.48.4:60016: {}
panic: runtime error: index out of range

goroutine 159 [running]:
panic(0x1c6d8e0, 0xc420014060)
	/home/travis/.gimme/versions/go1.7.1.linux.amd64/src/runtime/panic.go:500 +0x1a1
github.com/kubernetes/dashboard/src/app/backend/integration/metric/heapster.heapsterClient.ithResourceDownload.func1(0x2cc5e60, 0xc4204555f0, 0x1e95537, 0x4, 0x1e97a5e, 0x6, 0xc42077ff80, 0x4, 0x4, 0xc420aa5050, ...)
	/home/travis/build/kubernetes/dashboard/.tmp/backend/src/github.com/kubernetes/dashboard/src/app/backend/integration/metric/heapster/client.go:171 +0x5a5
created by github.com/kubernetes/dashboard/src/app/backend/integration/metric/heapster.heapsterClient.ithResourceDownload
	/home/travis/build/kubernetes/dashboard/.tmp/backend/src/github.com/kubernetes/dashboard/src/app/backend/integration/metric/heapster/client.go:177 +0x109
```

##### Comments
Similar to #2199 . Tried on both k8s `1.6.x` and `1.7.x`; same result!

**Comment by cnwalker** [comment]
I get this for Cluster but not nodes on k8s `1.6.x`

**Comment by idealhack** [comment]
Duplicate of #2187 and fixed by #2193
